packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md

```diff
@@ -0,0 +1,247 @@
+      }
+    },
+    "userProfile": {
+      "firstName": "test",
+      "lastName": "user",
+      "login": "test.user@okta.com",
+      "userId": "00uyxxSknGtK8022w0g3"
+    },
+    "messageProfile": {
+      "msgTemplate": "(HOOK)Your code is 11111",
+      "phoneNumber": "9876543210",
+      "otpExpires": "2022-01-28T21:48:34.321Z",
+      "deliveryChannel": "SMS",
+      "otpCode": "11111",
+      "locale": "EN-US",
+      "eventAction": "com.okta.user.telephony.pre-enrollment"
+    }
+  }
+}
+```
+
+## Sample JSON payloads of responses
+
+This section provides example JSON payloads for the supported operations.
+
+### Sample response for successful OTP delivery
+
+```json
+{
+  "commands":[
+    {
+      "type":"com.okta.telephony.action",
+      "value":[
+        {
+          "status":"SUCCESSFUL",
+          "provider":"VONAGE",
+          "transactionId":"SM49a8ece2822d44e4adaccd7ed268f954",
+          "transactionMetadata":"Duration=300ms"
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Sample response for failed OTP delivery
+
+```json
+{
+  "error":{
+    "errorSummary":"Failed to deliver SMS OTP to test.user@okta.com",
+    "errorCauses":[
+      {
+        "errorSummary":"Provider could not deliver OTP",
+        "reason":"The content of the message is not supported",
+        "location":"South Africa"
+      }
+    ]
+  }
+}
+```
+
+## Timeout behavior
+
+After receiving the Okta request, if there is a response timeout, the Okta process flow proceeds with trying to send the
+OTP using Okta's telephony provider(s). See [Troubleshooting](#troubleshooting).
+
+## Troubleshooting
+
+This section covers what happens when a telephony inline hook flow fails either due to the external inline hook service returning an error object or not returning a successful response.
+
+> **Note:** Administrators can use the [Okta System Log](/docs/reference/api/system-log/) to view errors. See the [Troubleshooting](/docs/concepts/inline-hooks/#troubleshooting) section in the Inline Hooks concept piece for more information on the events related to Inline Hooks that the Okta System Log captures.
+
+- When there is a communication failure with the external service, a timeout for example, the inline hook operation is skipped.
+  OTP is delivered to requester using Okta's telephony provider(s).
+
+  **Who can see this error?** Administrators
+
+- When the external service returns a response with any other HTTP status code besides `200`, the inline hook operation is skipped.
+  OTP is delivered to requester using Okta's telephony provider(s).
+
+  **Who can see this error?** Administrators
+
+- When the external service returns an error object in the response, the telephony inline hook flow fails with no OTP delivered using the hook.
+
+  **Who can see this error?** Administrators, developers, and end users.
+
+- When a hook response is malformed or could not be mappeed to expected API response, the inline hook operation is skipped.
```

Comment:
```suggestion
- When a hook response is malformed or couldn't be mapped to the expected API response, the inline hook operation is skipped, and the OTP is delivered to the requester using Okta's telephony provider(s). There may be scenarios where the requester may receive duplicate SMS/CALL messages because of the external service and Okta provider(s) both attempting to deliver the OTP.
```